samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -213,27 +195,27 @@ private void generateNewJobModel(List<String> processors) {
     String currentJMVersion  = zkUtils.getJobModelVersion();
     String nextJMVersion;
     if (currentJMVersion == null) {
-      log.info("pid=" + processorId + "generating first version of the model");
+      LOGGER.info("pid=" + processorId + "generating first version of the model");
       nextJMVersion = "1";
     } else {
       nextJMVersion = Integer.toString(Integer.valueOf(currentJMVersion) + 1);
     }
-    log.info("pid=" + processorId + "generating new model. Version = " + nextJMVersion);
+    LOGGER.info("pid=" + processorId + "generating new model. Version = " + nextJMVersion);
 
     List<String> containerIds = new ArrayList<>(currentProcessorsIds.size());
     for (String processorPid : currentProcessorsIds) {
       containerIds.add(processorPid);
     }
-    log.info("generate new job model: processorsIds: " + Arrays.toString(containerIds.toArray()));
+    LOGGER.info("generate new job model: processorsIds: " + Arrays.toString(containerIds.toArray()));
 
     JobModel jobModel = JobModelManager.readJobModel(this.config, Collections.emptyMap(), null, streamMetadataCache,
         containerIds);
 
-    log.info("pid=" + processorId + "Generated jobModel: " + jobModel);
+    LOGGER.info("pid=" + processorId + "Generated jobModel: " + jobModel);
 
     // publish the new job model first
     zkUtils.publishJobModel(nextJMVersion, jobModel);
-    log.info("pid=" + processorId + "published new JobModel ver=" + nextJMVersion + ";jm=" + jobModel);
+    LOGGER.info("pid=" + processorId + "published new JobModel ver=" + nextJMVersion + ";jm=" + jobModel);
```

Comment:
I know it is existing code. Please remove the job model from this log line. We don't need it twice in the log.